src/database/DataTypes/Location.js

```diff
@@ -5,13 +5,105 @@
 
 import Realm from 'realm';
 
+import { extractBreaches } from '../../utilities/modules/vaccines';
+import { getTotal } from '../utilities';
+
 export class Location extends Realm.Object {
-  // TODO
-  // getBreaches(database) <- ('SensorLogs') filter this ID and isInBreach = true
-  // getBatches ?
+  // TODO remove data object
+  data = {
+    isInBreach: false,
+    totalStock: 16,
+    currentTemperature: '5.2',
+    temperatureExposure: { minTemperature: 2.5, maxTemperature: 10 },
+    temperaturePoints: {
+      maxLine: [
+        { date: 'Feb 23', temp: 5 },
+        { date: 'Feb 24', temp: 8 },
+        { date: 'Feb 25', temp: 8.5 },
+        { date: 'Feb 26', temp: 9 },
+        { date: 'Feb 27', temp: 10 },
+        { date: 'March 1', temp: 7 },
+        { date: 'March 2', temp: 5 },
+        { date: 'March 3', temp: 4 },
+        { date: 'March 4', temp: 5 },
+        { date: 'March 5', temp: 6 },
+      ],
+      minLine: [
+        { date: 'Feb 23', temp: 2.5 },
+        { date: 'Feb 24', temp: 5 },
+        { date: 'Feb 25', temp: 5.7 },
+        { date: 'Feb 26', temp: 7.5 },
+        { date: 'Feb 27', temp: 8 },
+        { date: 'March 1', temp: 6 },
+        { date: 'March 2', temp: 3.4 },
+        { date: 'March 3', temp: 3 },
+        { date: 'March 4', temp: 2.5 },
+        { date: 'March 5', temp: 4 },
+      ],
+      hazards: [{ date: 'Feb 27', temp: 10, onClick: () => console.log('CLICKED HAZARD 4') }],
+      minTemp: 2,
+      maxTemp: 8,
+    },
+  };
 
   get isFridge() {
-    return this.locationType && this.locationType.description.toLowerCase() === 'fridge';
+    const { locationType: { description } = { description: '' } } = this;
+    return description.toLowerCase() === 'fridge';
+  }
+
+  getTemperaturePoints() {
```

Comment:
I think removing?

Reply:
Also just removed in: https://github.com/openmsupply/mobile/compare/%23874-Vaccine-Module-Page...%23874-Refinements]-Vaccine-Modul-Page, getter still used in PR